### ISSUE TITLE
Render / load file from actual pdf file

### ISSUE THREAD

**Issue by cyrilchapon** [diegomura/react-pdf#854]
Hey;

I feel stupid but I can't find this piece of information...

Is that lib able to display in-browser an actual .pdf (maybe downloaded from blob or something) ?
Maybe another well-known lib using this renderer ?

Thanks :)

**Comment by aymeric-giraudet** [comment]
I think what you're looking for is this : https://github.com/wojtekmaj/react-pdf
Same name, different purpose :)

Unfortunately this library's PDFViewerComponent is for rendering what's in a Document component only.

**Comment by cyrilchapon** [comment]
Hey @aymeric-giraudet ; thanks for the reference.

I'm using wojtekmaj/react-pdf in production right now indeed; and it's working like a charm.

For the sake of feedback; I can tell that when I landed to this repository (through SEO), "discovering the purpose" was unclear to me. I might be stupid, and I'm pretty the use-case it tacles is very popular; but I struggled at understanding it was not what I was looking for. If this can helps doc a little bit 🙂
